docs/components/accordion/README.md

```diff
@@ -119,6 +119,19 @@
       "version": "0.1.0"
     }
   ],
+  "see_also": [
+    {
+      "text": "See Also",
+    },
+    {
+      "text": "Lists",
+      "href": "/components/lists/"
```

Comment:
This won't work live since it has a different base path. Need to use relative paths for links --> ../lists/

Reply:
Fixed.